docs/plans/2026-05-13-chore-enroll-catalog-candidates-plan.md

```
@@ -0,0 +1,122 @@
+
+- [ ] `bun ./scripts/lint/no-duplicate-deps.ts` produces **0 CATALOG CANDIDATES** for third-party packages
+- [ ] **0 CATALOG VIOLATIONS** and **0 VERSION MISMATCHES** (must remain at zero)
+- [ ] `bun run check-types` exits 0 after changes
+- [ ] `bun install` re-run to sync lockfile after package.json edits
+
+## Pattern
+
+**Two-step for each candidate:**
+
+1. Add an entry to the `"catalog"` object in root `package.json`:
+   ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)